pkg/reconciliation/reconcile_racks.go

```diff
@@ -617,11 +618,13 @@ func (rc *ReconciliationContext) CheckPodsReady(endpointData httphelper.CassMeta
 		return result.RequeueSoon(5)
 	}
 
-	needsMoreNodes, err := rc.startAllNodes(endpointData)
+	// step 4 - make sure all nodes are ready
+
+	nodeNotReady, err = rc.startAllNodes(endpointData)
```

Comment:
Amm, why the same boolean for special process of restarting one node per rack and for rest of the starts? I don't see the point of this refactor. Yet most of the other parts still have their own booleans.

Reply:
I reused the same boolean because they really mean exactly the same thing: there is at least one pod not ready. But ok, if you want each step to have its own boolean, let's do it.